<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RroDialog</class>
 <widget class="QDialog" name="RroDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEditFN"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Владелец:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboBoxVlasnik"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>ЗН:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEditZN"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>ФН:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEditTerminal"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Адрес:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="lineEditAdress"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Терминал:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Модель РРО:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboBoxModel"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Версия ПО:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="comboBoxVersion"/>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="toolButtonVlasnik">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="toolButtonModel">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="toolButtonVersion">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxVlasnik</tabstop>
  <tabstop>lineEditZN</tabstop>
  <tabstop>lineEditFN</tabstop>
  <tabstop>comboBoxModel</tabstop>
  <tabstop>comboBoxVersion</tabstop>
  <tabstop>lineEditTerminal</tabstop>
  <tabstop>lineEditAdress</tabstop>
  <tabstop>toolButtonVlasnik</tabstop>
  <tabstop>toolButtonModel</tabstop>
  <tabstop>toolButtonVersion</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
